ibtk/src/solvers/impls/SCPoissonHypreLevelSolver.cpp

```diff
@@ -65,6 +66,16 @@ static Timer* t_solve_system_hypre;
 static Timer* t_initialize_solver_state;
 static Timer* t_deallocate_solver_state;
 
+// HYPRE can use 64bit indices, but SAMRAI IntVectors are always 32 - add
```

Comment:
Should we pull this up to a helper header since it now shows up in a few places?

Reply:
I'd rather not since these two files are the only places it shows up (and we don't interface with hypre anywhere else) - having an extra header `hypre_compatibility.h` or whatever is more work than copying and pasting this seven-line function once.

Reply:
That's fine --- I was thinking about using an existing header like `IndexUtilities.h` or `ibtk_utilities.h`.